tox_crypto/src/lib.rs

```diff
@@ -4,9 +4,6 @@
 
 pub use sodiumoxide::randombytes::randombytes_into;
```

Comment:
What about randombytes_into? Why isn't it replaced with rand?

Reply:
Because I missed it :)